apps/e2e/cypress/e2e/back-office-applications/s51Advice.spec.js

```diff
@@ -55,6 +55,7 @@ describe('Section 51 Advice', () => {
 				},
 				titlefirst
 			);
+			s51AdvicePage.publishS51WithoutWelshFields();
```

Comment:
don't you need to run this behind a flag? it would fail if on a Welsh case

Reply:
Runs only when excludeWales: true

Reply:
no need to worry about flag